mu4/scores/view/newscoremodel.cpp

```diff
@@ -0,0 +1,95 @@
+#include "newscoremodel.h"
+
+#include "actions/actiontypes.h"
+
+#include "ret.h"
+#include "log.h"
+
+using namespace mu::scores;
+using namespace mu::actions;
+using namespace mu::domain::notation;
+
+NewScoreModel::NewScoreModel(QObject* parent)
+    : QObject(parent)
+{
+}
+
+bool NewScoreModel::create()
+{
+    ScoreCreateOptions scoreOptions;
+    scoreOptions.title = m_title;
+    scoreOptions.composer = m_composer;
+    scoreOptions.templatePath = io::pathToQString(
+        globalConfiguration()->sharePath() + "/templates/02-Choral/05-SATB_Closed_Score_+_Organ.mscx");
```

Comment:
Why this template as the default score?

Reply:
Creating a new score is done just for display. In the future, there will be a New Score Wizard, only in qml. This is a different task. And there will be a choice of template and other properties for the new score.

Reply:
I suggest adding: `TODO Temporary solution`